src/crypto/siphash.h

```diff
@@ -48,6 +48,20 @@ class CSipHasher
  *      .Finalize()
  */
 uint64_t SipHashUint256(uint64_t k0, uint64_t k1, const uint256& val);
-uint64_t SipHashUint256Extra(uint64_t k0, uint64_t k1, const uint256& val, uint32_t extra);
+
+class PresaltedSipHasher
```

Comment:
In commit "optimization: Introduce PresaltedSipHasher for repeated hashing" (8568340be2c679713703edc3d418272ac78d1147)

Can description of this class from commit description be added here so there is some documentation for this. Also not sure if the distinction between salted/presalted is significant, but it might be good to be more consistent if both are correct.